CHANGELOG.md

```diff
@@ -16,8 +20,24 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Eric Amorde](https://github.com/amorde)
   [#7093](https://github.com/CocoaPods/CocoaPods/pull/7093)
 
+* Show warning when Pod source uses unencrypted HTTP  
+  [KrauseFx](https://github.com/KrauseFx)
+  [#7238](https://github.com/CocoaPods/CocoaPods/issues/7238)
+
+* Show warning when SDK provider tries to push a version with an unencrypted HTTP source  
+  [KrauseFx](https://github.com/KrauseFx)
+  [#7250](https://github.com/CocoaPods/CocoaPods/pull/7250)
+
 ##### Bug Fixes
 
+* Deduplicate output path file names for resources and frameworks  
+  [Eric Amorde](https://github.com/amorde)
+  [#7259](https://github.com/CocoaPods/CocoaPods/issues/7259)
+
+* Allow installation of a pod with its own Swift version on multiple targets  
+  [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
+  [#7261](https://github.com/CocoaPods/CocoaPods/pull/7261)
```

Comment:
It seems this changes were unintentional - may need to rebase against master

Reply:
@amorde can you send me an email saying "Hi" (my emails in on profile, want to invite you to the CocoaPods slack ) 👍

Reply:
> It seems this changes were unintentional - may need to rebase against master

yeap